hope_horizon/backend/views/group_user_view.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from backend.serializers import GroupUserSerializer
+from backend.models import CustomGroup, GroupUser
+from rest_framework.permissions import IsAuthenticated
+from django.db.models import Q
+
+class GroupUserViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated]
+    serializer_class_group_user = GroupUserSerializer
+
+    def list(self, request):
+        user = request.user
+        id = request.query_params.get("group_id", None)
+        try:
+            group = CustomGroup.objects.get(pk=id)
+        except CustomGroup.DoesNotExist:
+            return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
can you please change all your Responses so that they have the same format as Stefan and mine:
Example: return Response({"detail": "Owned parameter is required"}, status=status.HTTP_400_BAD_REQUEST)